slicedimage/backends/_caching.py

```diff
@@ -1,31 +1,37 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import os
+import io
+
+from diskcache import Cache
 
 from ._base import Backend
 
+size_limit = 5e9
```

Comment:
There aren't really true constants in python, but "constants" in python are generally given all caps names, i.e., SIZE_LIMIT